python/tank_vendor/shotgun_authentication/login_dialog.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) 2013 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+"""
+Login dialog for authenticating to a Shotgun server.
+"""
+
+from .ui import resources_rc
+from .ui import login_dialog
+from . import session_cache
+from .errors import AuthenticationError
+from .ui.qt_abstraction import QtGui, QtCore
+
+
+class LoginDialog(QtGui.QDialog):
+    """
+    Dialog for getting user crendentials.
+    """
+    def __init__(self, title, is_session_renewal, hostname="", login="", fixed_host=False, http_proxy=None, parent=None):
```

Comment:
Do we really need to expose the title parameter? I suggest hard coding it in the class.